app/javascript/components/student/editor/ResultsTab.tsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react'
+import { Tab } from './Tab'
+import { TabIndex } from '../Editor'
+import { GraphicalIcon } from '../../common/GraphicalIcon'
+
+export const ResultsTab = () => (
+  <Tab index={TabIndex.RESULTS}>
+    <GraphicalIcon icon="test-results" />
+    <span data-text="Results">Results</span>
+  </Tab>
```

Comment:
All tabs have this format layout, (ie an icon and some text in a span with an attr) so the contents of this could probably be safely moved into the `<Tab>` class. We also use that Tab class in other places, so it could probably be a Common component.